app.py

```diff
@@ -0,0 +1,123 @@
+            if username in user.keys():
+                return jsonify({"message": "User already exists"})
+
+        new_user = User(firstname, username, password)
+        Users.append(new_user)
+        response = {"message": "User created successfully"}
+        return (jsonify(response)), 201
+
+@app.route('/api/auth/login', methods=['POST'])
+def login_user():
+    if request.method == 'POST':
+        data = request.get_json()
+        username = data.get("username")
+        password = data.get("password")            
+        details = {new_user.username: new_user.password for new_user in Users}
+        if username in details.keys():
+            access_token = create_access_token(identity=username)
+            response = {
+                "message": "You are successfully logged in",
+                "access_token": access_token
+            }
+            return (jsonify(response)), 201
+        else:
+            return jsonify ({'message': 'User does not exist. Proceed to register'}), 500
+            
+@app.route('/api/register/business', methods=['GET', 'POST'])
+@jwt_required
+def register_business():
+    if request.method == 'POST':
+        current_user = get_jwt_identity()
+        data = request.get_json()
+        businessname = data.get("businessname")
+        description = data.get("description")
+        location = data.get("location")
+
+        if businessname.strip() == "": 
+            return jsonify({"message": "Please input a businessname value"}), 406
+        elif description.strip() == "":
+            return jsonify({"message": "Please input a description value"})
+        elif location.strip() == "":
+            return jsonify({"message": "Please input location a value"})
+        else:
+            business = {new_business.businessname: new_business.description for new_business in Businesses}
+            if businessname in business.keys():
+                return jsonify({"message": "Business already exists"})
+
+        new_business = Business(businessname, description, location)
+        Businesses.append(new_business)
+        response = {"message": "Business created successfully",
+                    "Business created by": current_user
+            }
+        return (jsonify(response)), 201
+
+    else:
+        if request.method == 'GET':
+            return jsonify(Businesses), 200
+
+@app.route('/api/businesses/review', methods=['GET', 'POST'])
+@jwt_required
+def register_review():
+    if request.method == 'POST':
+        current_user = get_jwt_identity()
+        data = request.get_json()
+        title = data.get("title")
+        description = data.get("description")
+
+        if title.strip() == "": 
+            return jsonify({"message": "Please input a title value"}), 406
+        else:
+            if description.strip() == "":
+                return jsonify({"message": "Please input a description value"})
+    

```

Comment:
Why does a review have a title and description? Shouldn't a user just be able to submit a review without having to enter so many details?